config/routes.rb

```diff
@@ -1,8 +1,16 @@
 Rails.application.routes.draw do
   scope "(:locale)", locale: /en|vi/ do
     root to: "static_pages#home"
-    get "static_pages/product"
-    get "static_pages/about"
-    get "static_pages/contact"
+    get "/product", to: "static_pages#product"
+    get "/about", to: "static_pages#about"
+    get "/contact", to: "static_pages#contact"
+#    get "users/new"
```

Comment:
bỏ hết cmt đi nha!

Reply:
cmt là để tý t làm sign in sign out tiếp nên cứ kệ nó, mấy lỗi kia nếu ok r thì t làm tiếp sign in sign out luôn.